GOVERNANCE.md

```diff
@@ -64,10 +64,6 @@ A guide for Collaborators is maintained in
 ## CTC Membership
 
 CTC seats are not time-limited.  There is no fixed size of the CTC.
-However, the expected target is between 6 and 12, to ensure adequate
-coverage of important areas of expertise, balanced with the ability to
-make decisions efficiently.
```

Comment:
There's always the option of some "including, but not limited to, X, Y, and Z" language....